src/components/AuthModal/AuthModal-container.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+
+import AuthModalView from './AuthModal-view';
+
+import { AuthModalProps, User } from './types'
+
+const AuthModal = ({ initLogin }: AuthModalProps) => {
+  const [open, setOpen] = useState(false);
+  const [forgetPassword, setForgetPassword] = useState(false);
+  const [rememberMe, setRememberMe] = useState(true);
+  const [user, setUser] = useState({
+    email: '',
+    password: ''
+  });
+
+  const handleOpen = () => {
+    setOpen((prevState => !prevState));
+  };
+
+  const handleClose = () => {
+    setOpen((prevState => !prevState));
+    setForgetPassword(false);
+    setRememberMe(true);
+    setUser({
+      email: '',
+      password: ''
+    })
+  };
+
+  const handleSwitchForms = () =>  {
+    setForgetPassword(prevState => !prevState);
+  };
+  
+  const handleRememberMe = () => {
+    setRememberMe(prevState => !prevState);
```

Comment:
Może handleOpen, handleSwitchForms, handleRememberMe, handleLogin skróć do jednej linijki?

Reply:
hmm a jak mam state `rememberMe` to jak inaczej zadeklarować funkcje bez `handle` ? 🤔